apiary.apib

```diff
@@ -991,9 +991,247 @@ Outputs the list of peers that were removed.
 
 # Group log
 
-## level
+Interact with the daemon log output
 
-## tail
+'ipfs log' contains utility commands to affect or read the logging
+output of a running daemon.
+
+This command cannot be called directly.
+
+## level [POST /log/level{?arg1,arg2}]
+'ipfs log level' is a utility command used to change the logging
+output of a running daemon.
+
++ Parameters
+    + arg1 (string) - The subsystem logging identifier. Use `all` for all subsystems.
+        + Members
+            - bitstwap
+            - blockstore
+            - dht
+            - merkledag
+            - all
+    + arg2 (enum[string])
```

Comment:
Maybe sth like this ` - The log level, with `debug` the most verbose and `panic` the least verbose.`
Also are these two args optional or required, if they are optional what is the default value :P